hippunfold/config/snakebids.yml

```diff
@@ -356,37 +356,28 @@ xfm_identity_itk: resources/etc/identity_xfm_itk.txt
 
 #templates enabled for template-based segmentation are here
 # TODO: should also perhaps include modalities avaialble, and any custom crop_native_res settings
-template_based_segmentation:
+template_available_hemis:
```

Comment:
Why not put this inside the template_files dict to keep things together? I can do that in my edits..

Reply:
just made that edit -- generally trying to keep things hierarchical in the config (will make anotehr issue to do this en-masse, as the config right now is really messy with so many different top-level keys (which don't remain sorted in the nice way that we have them when deployed to an output folder)..